doc/ins_sign_tx.md

```diff
@@ -64,7 +64,33 @@ Initializes signing request.
 | number of tx withdrawals                  | 4 | Big endian |
 | number of tx witnesses                    | 4 | Big endian |
 
-### 2 - Set UTxO inputs
+### 2 - Auxiliary data
+
+Optional.
+
+|Field|Value|
+|-----|-----|
+|  P1 | `0x08` |
+|  P2 | (unused / see [Catalyst Registration](ins_sign_catalyst_registration.md)) |
+
+**Data for AUX_DATA_TYPE_ARBITRARY_HASH**
+
+Ledger cannot parse and display generic auxiliary data in full because their structure is too loose and their memory footprint potentially too big.
+So only the hash is transferred and displayed and the user has to use other means of verification that the hash is correct.
+
+|Field| Length | Comments|
+|-----|--------|--------|
+| Auxiliary data type | 1 | |
```

Comment:
I think we should mention the value in the last column (cf. `CERTIFICATE_TYPE_STAKE_REGISTRATION=0x00` elsewhere). Also below.